src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonIncentiveLevelService.kt

```diff
@@ -32,6 +32,13 @@ class PrisonIncentiveLevelService(
   private val incentiveLevelRepository: IncentiveLevelRepository,
   private val prisonIncentiveLevelRepository: PrisonIncentiveLevelRepository,
 ) {
+  /**
+   * Returns all incentive levels for given prison, including inactive ones, along with associated information, in globally-defined order
+   */
+  suspend fun getAllPrisonIncentiveLevels(prisonId: String): List<PrisonIncentiveLevelDTO> {
+    return prisonIncentiveLevelRepository.findAllByPrisonId(prisonId).toListOfDTO()
```

Comment:
It's an interesting one at prison level 🤔 

If a `(prisonId, levelCode)` row is not present for whatever reason it wouldn't be in the list/returned.

This will only include levels which were at some point added to a prison, including the ones that were deactivated. 
This could be a bit unpredictable.

I think it makes a lot of sense for the endpoint returning the "global" list of Incentive levels to optionally return all of the levels.

However what's the use case for the prison-level list endpoint to return "all" levels?